doc/developers/bug_triaging.rst

```diff
@@ -0,0 +1,146 @@
+    Stack Overflow instead.
+
+  - **close duplicate issues**, but only after checking that they are
+    indeed duplicate. Ideally, the original submitter moves the
+    discussion to the older, duplicate issue
+
+  - **close issues that cannot be replicated**, after leaving time (at
+    least a week) to add extra information
+
+:ref:`Saved replies <saved_replies>` are useful to gain time and yet be
+welcoming and polite when triaging.
+
+
+.. topic:: Closing issues: a tough call
+
+    When uncertain on whether an issue should be closed or not, it is
+    best to strive for consensus with the original poster, and possibly
+    to seek relevant expertise. However, when the issue is a usage
+    question, or when it has been considered as unclear for many years it
+    should be closed.
+
+A typical workflow for triaging issues
+----------------------------------------
+
+The following workflow [*]_ is a good way to approach issue triaging:
+
+1. Thank the reporter for opening an issue
+
+   The issue tracker is many people’s first interaction with the
+   scikit-learn project itself, beyond just using the library. As such,
+   we want it to be a welcoming, pleasant experience.
+
+2. Is this a usage question? If so close it with a polite message
+   (:ref:`here is an example <saved_replies>`).
+
+3. Is the necessary information provided?
+
+   If crucial information (like the version of scikit-learn used), is
+   missing feel free to ask for that and label the issue with "Needs
+   info".
+
+4. Is this a duplicate issue?
+
+   We have many open issues. If a new issue seems to be a duplicate,
+   point to the original issue. If it is a clear duplicate, or consensus
+   is that it is redundant, close it. Make sure to still thank the
+   reporter, and encourage them to chime in on the original issue, and
+   perhaps try to fix it.
+
+   If the new issue provides relevant information, such as a better or
+   slightly different example, add it to the original issue as a comment
+   or an edit to the original post.
+
+
+5. Make sure that the title accurately reflects the issue. Edit it
+   yourself if it's not clear.
+
+6. Is the issue minimal and reproducible?
+
+   For bug reports, we ask that the reporter provide a minimal
+   reproducible example. See
+   https://matthewrocklin.com/blog/work/2018/02/28/minimal-bug-reports
+   for a good explanation. If the example is not reproducible, or if
+   it's clearly not minimal, feel free to ask the reporter if they can
+   provide and example or simplify the provided one. Do acknowledge that
+   writing minimal reproducible examples is hard work. If the reporter
+   is struggling, you can try to write one yourself.
+
+   If a reproducible example is provided, but you see a simplification,
+   edit the original post with your simpler reproducible example.
+
+4. If a reproducible example can't be provided, add the “Bug: triage”
+   label.
```

Comment:
The "Bug: triage" is the default for issues opened as "Bug" template. The issue here is more about when the label "Bug: triage" can be replaced with the "Bug" one?